src/main/java/org/fundacionjala/at15/spaceinvaders/Board.java

```diff
@@ -39,20 +34,6 @@ public void keyReleased(KeyEvent e) {
         setFocusable(true);
     }
 
-    public List<Alien> getAliens() {
-        return this.aliens;
-    }
-
-    public void createAliens(int rows, int columns) {
-        for (int yIndex = 0; yIndex < rows; yIndex++) {
-            for (int xIndex = 0; xIndex < columns; xIndex++) {
-                Alien alien = new Alien(
-                        ALIEN_INIT_X + ALIEN_SEPARATION * xIndex,
-                        ALIEN_INIT_Y + ALIEN_SEPARATION * yIndex, ALIEN_WIDTH, ALIEN_HEIGHT);
-                this.aliens.add(alien);
-            }
-        }
-    }
     public static void pause() {
```

Comment:
- [ ] We need to find a way to update all renders at the same time without this method.